app/src/main/AndroidManifest.xml

```diff
@@ -28,7 +28,6 @@
                 android:hardwareAccelerated="true"
                 android:allowTaskReparenting="true"
                 android:windowSoftInputMode="adjustResize"
-                android:alwaysRetainTaskState="true"
```

Comment:
Could you explain, why you did remove this?

Reply:
By default, Android removes application state after 30 minutes. There's no reason to disable this behaviour for SupProd, and furthermore, I don't think it even works with WebView apps, so this should have no effect

Reply:
Thanks for the explanation!